sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/csv/CSVSuite.scala

```diff
@@ -477,15 +478,15 @@ class CSVSuite extends QueryTest with SharedSQLContext with SQLTestUtils {
     val options = Map(
       "header" -> "true",
       "inferSchema" -> "true",
-      "dateFormat" -> "dd/MM/yyyy hh:mm")
+      "dateFormat" -> "dd/MM/yyyy HH:mm")
```

Comment:
This was wrong because the times as below:

initial CSV data is as below:

```
26/08/2015 18:00
27/10/2014 18:30
28/01/2016 20:00
```

The hour is based on 24 hour-based one but `hh` means 12 hour-based one (See https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html)